CONTRIBUTING.rst

```diff
@@ -7,122 +7,69 @@ Contributing
 Contributions are welcome, and they are greatly appreciated! Every little bit
 helps, and credit will always be given.
 
-You can contribute in many ways:
 
-Types of Contributions
-----------------------
-
-Report Bugs
-~~~~~~~~~~~
-
-Report bugs at https://github.com/esss/alfasim_sdk/issues.
-
-If you are reporting a bug, please include:
-
-* Your operating system name and version.
-* Any details about your local setup that might be helpful in troubleshooting.
-* Detailed steps to reproduce the bug.
-
-Fix Bugs
-~~~~~~~~
-
-Look through the GitHub issues for bugs. Anything tagged with "bug" and "help
-wanted" is open to whoever wants to implement it.
+Get Started!
```

Comment:
We should add a blurb about how to develop `alfasim_sdk` inside ESSS.

Reply:
Commit https://github.com/ESSS/alfasim-sdk/pull/35/commits/0ee145fe31cba6ec5ee22e099ea0fd124f877976 add some instructions, could review it?